kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/NetworkAdminServiceImpl.java

```diff
@@ -88,6 +89,8 @@
 
 public class NetworkAdminServiceImpl implements NetworkAdminService, EventHandler {
 
+    private static final String ACCESS_POINT_INTERFACE_SUFFIX = "_ap";
+
```

Comment:
Since this constant is also in the NetworkServiceImpl, maybe it's better to define it only once.